lib/android_build/app/build.gradle

```diff
@@ -30,6 +30,10 @@ android {
             version "3.10.2"
         }
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }

```

Comment:
Will this means minimum JAVA SE 8 would be required to build 1ds-sdk for android, even if okhttp is not used as HTTPClient ?

Reply:
Current Android Studio uses Java 11, this tells it to use 1.8 which is what it was built with